code/src/components/AddTask.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/tasks'
+import 'css/buttons.css'
+import 'css/task.css'
+
+export const AddTask = () => {
+  const [text, setText] = useState('')
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(tasks.actions.addTodo(text))
+    setText('')
+  }
```

Comment:
to make this even better you could check so that the text variable isn't empty before saving it as a todo? right now you could create empty todos.
```
const handleSubmit = (event) => {
  event.preventDefault()
  
  if (text) {
    dispatch(todo.actions.addItem(text))
    setText('')
  }
}
```